src/DIRAC/Resources/Storage/GFAL2_SRM2Storage.py

```diff
@@ -30,8 +30,13 @@
 class GFAL2_SRM2Storage(GFAL2_StorageBase):
     """SRM2 SE class that inherits from GFAL2StorageBase"""
 
-    _INPUT_PROTOCOLS = ["file", "root", "srm", "gsiftp"]
-    _OUTPUT_PROTOCOLS = ["file", "root", "dcap", "gsidcap", "rfio", "srm", "gsiftp"]
+    _INPUT_PROTOCOLS = [
+        "file",
+        "gsiftp",
+        "root",
+        "srm",
+    ]
+    _OUTPUT_PROTOCOLS = ["gsiftp", "root", "srm"]
```

Comment:
No, not `file` anymore. 
The reason is because this list of protocol was only ever used as base line to get a TURL (`TURL_PROTOCOLS`), but was superbly ignored for downloading or putting a file (`TURL_3RD_PARTY_PROTOCOLS`. Don't ask.....).
So now that we also use this list for that, having file by default would only work if you are on a WN where the file system is mounted. 
So it is better to remove it from the default, and explicitly set it in the CS when appropriate.
Am I making sense ?

Reply:
Ahh, I thought this meant writing to "file:///foo/bar"